server/satnogs-web.py

```diff
@@ -1,5 +1,7 @@
 #!/usr/bin/env python3
 from app import app
 
+app.config['SECRET_KEY'] = 'the earth is flat'
+
 if __name__ == '__main__':
-    app.run(debug=True, port=5000)
+    app.run(debug=True, port=8080)
```

Comment:
Uhh, why? You make me to reconfigure my IDE.

Reply:
I have other projects that are being run locally. All of them use 8080. But if it's a problem, I can withdraw this change.